### PR TITLE
Updated to run on localnet

### PR DESCRIPTION
**Pull request by ASCorreia**

### CODE REVIEW

File: tokens/nft-minter/anchor/tests/test.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('NFT Minter', () => {
 
     const transactionSignature = await program.methods
       .mintNft(metadata.name, metadata.symbol, metadata.uri)
-      .accounts({
+      .accountsPartial({
```
Comment (jacobcreech) [inline tokens/nft-minter/anchor/tests/test.ts:28]: Is this because of anchor 0.30.x's pda auto resolution?

Comment (ASCorreia) [reply tokens/nft-minter/anchor/tests/test.ts:28]: Yes. Added just to be explicit, in the example, on the account to pass in the instruction

**Comment by heyAyushh** [comment]
some of those programs are missing pnpm-lock.yaml in tests and hence are failing.

For fix, make sure to install packages using https://pnpm.io/ and commit the yaml file 🫡

More in contributing guidelines 
https://github.com/solana-developers/program-examples/blob/main/CONTRIBUTING.md#contributing-code-examples

**Comment by ASCorreia** [comment]
> some of those programs are missing pnpm-lock.yaml in tests and hence are failing.
> 
> For fix, make sure to install packages using https://pnpm.io/ and commit the yaml file 🫡
> 
> More in contributing guidelines https://github.com/solana-developers/program-examples/blob/main/CONTRIBUTING.md#contributing-code-examples

The programs already contained a pnpm-lock.yaml that is now up to date with package.json

**Pull request closed without merging**